monitors/filterWords.js

```diff
@@ -0,0 +1,42 @@
+exports.conf = {
+  enabled: true,
+  // Change to false if the filter should not ignore other bots
+  ignoreBots: true,
+  // Change to true if the filter should ignore selfBots
+  ignoreSelf: false,
```

Comment:
Well, unless you have an echo command, you might not want your own bot to check its own messages.